packages/foreman/nodejs-invariant/nodejs-invariant.spec

```diff
@@ -15,6 +15,7 @@ Source0: https://registry.npmjs.org/%{npm_name}/-/%{npm_name}-%{version}.tgz
 BuildRequires: %{?scl_prefix_nodejs}npm
 %else
 BuildRequires: nodejs-packaging
+BuildRequires: npm
```

Comment:
I really still wonder if this is needed. It shouldn't be, at least on EL7 and Fedora as far as I checked.